mangaki/mangaki/tests/test_reco.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+
+class RecoTest(TestCase):
+
+    def setUp(self):
+        User.objects.create_user(username='test', password='test')
+
+    def test_reco(self, **kwargs):
+        self.client.login(username='test', password='test')
+        reco_url = reverse('get-reco-algo-list', args=['svd', 'all'])
+        self.assertEqual(reco_url, '/data/reco/svd/all.json')
```

Comment:
It's okay to write models to disk and so on.

If you need seed data to do tests, it's also okay. Just let's prepare a minimal seed fixture for this test so that you can write reliable tests :).